### PR TITLE
view all articles

### PR DESCRIPTION
**Pull request by udochukwu**
**What does this PR do?**

- Users should be able to view all articles on Authors Haven

**Description of Task to be completed?**
- Setup view container for viewing all articles 

**What are the relevant pivotal tracker stories?**

- https://www.pivotaltracker.com/story/show/161790868

**Screenshots (if appropriate)**
image

**Questions**

- N/A

### CODE REVIEW

File: src/components/ArticleCard/ArticleCard.js [hunk 1]
```diff
@@ -16,15 +16,24 @@ export const ArticleCard = (props) => {
     articlePic = props.pic;
   }
   return (
-  <div>
-    <Card className={`${props.size} hoverable articleCard`}
-        header={<CardTitle image={articlePic}> <Link key={props.slug} className="capitalize" to={`/articles/${props.slug}`}>{props.title.substring(0, 50)}</Link> </CardTitle>}
-        actions={[<a href='#' key={props.slug}>Share</a>, <Link key={props.slug} to={`/articles/${props.slug}`}>View Article</Link>,
-        <span key={props.slug}>{moment(props.createdAt).format('D MMMM YYYY')}</span>, <span key={props.slug}>{props.author}</span>]}>
+    <div>
```
Comment (udochukwu) [inline src/components/ArticleCard/ArticleCard.js:19]: Nothing changed on this file. I adjusted the indentation.

File: src/components/DropDown/DropDown.js [hunk 2]
```diff
@@ -42,20 +43,20 @@ class DropDrown extends Component {
...
-          <li className="dropdown-list"><a href="#">Gadget</a></li>
-          <li className="dropdown-list"><a href="#">Programming</a></li>
-          <li className="dropdown-list"><a href="#">History</a></li>
-          <li className="dropdown-list"><a href="#">Fashion</a></li>
+          <li className="dropdown-list"><Link to={'#'}>Sport</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Medicine</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Gadget</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Programming</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>History</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Fashion</Link></li>
```
Comment (udochukwu) [inline src/components/DropDown/DropDown.js:59]: I changed all the anchor tags on the dropdown to a react-router link for consistency,

File: src/components/ArticleCard/ArticleCard.js [hunk 3]
```diff
@@ -16,15 +16,24 @@ export const ArticleCard = (props) => {
...
+      <Card className={`${props.size} hoverable articleCard`}
+        header={
+          <CardTitle image={articlePic}>
+            <Link key={props.slug} className="capitalize" to={`/articles/${props.slug}`}>
+              {props.title.substring(0, 50)}
+            </Link>
+          </CardTitle>}
+        actions={[
+          <a href='#' key={props.slug}>Share</a>,
```
Comment (onwuvic) [inline src/components/ArticleCard/ArticleCard.js]: use link not <a> tag

Comment (udochukwu) [reply src/components/ArticleCard/ArticleCard.js]: Alright thanks

**Review by oladimillion — COMMENTED** [review]
LGTM

**Comment by kampkelly** [comment]
LGTM

File: src/containers/ViewAllArticles/ViewAllArticles.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+   * @returns {jsx} - jsx
+   */
+  render() {
+    return (
+      <div>
+         <main>
+          <div className="site-container">
+          <Row>
+              <Col s={12} m={12} l={8} xl={9} className="column-1">
+                <Row>
```
Comment (s-akinrele) [inline src/containers/ViewAllArticles/ViewAllArticles.js:83]: indentation seems off

**Review by s-akinrele — CHANGES_REQUESTED** [review]
I had some comments

File: src/containers/ViewAllArticles/ViewAllArticles.js (hunk 4 above)
Comment (udochukwu) [reply src/containers/ViewAllArticles/ViewAllArticles.js:83]: @s-akinrele  I have fixed this issue.

**Review by s-akinrele — APPROVED** [review]

**Pull request merged**